plugins/base/src/test/resources/expect/signatureTest/out/-search.html

```diff
@@ -0,0 +1,23 @@
+<html>
```

Comment:
Are you sure this is signature specific file? If it does not, it should definitely be moved out of this PR